src/client.js

```
@@ -148,6 +148,14 @@ export default class DockerExecWebsocketClient extends EventEmitter {
     this.sendCode(msgcode.resume);
   }
 
+  resize (h, w) {
+    var buf = new Buffer(4);
```

Maybe we should throw an error if not started with `tty`